README.rst

```diff
@@ -29,8 +29,10 @@ Compatibility Notes
 +-------------------+-----------------------+
 |      Nutmeg       |         >= 6.2        |
 +-------------------+-----------------------+
+|       Olive       |         >= 8.0        |
```

Comment:
Maybe setting an upper bound to lilac is a good idea because we are still not testing with Django 22. What do you think?